views/analysis.py

```diff
@@ -809,6 +810,10 @@ def analysis_json(request, analysis_id):
             'analysis_id': analysis_id,
             'impact_layer_id': analysis.impact_layer_id
         }
+
+        if analysis.impact_layer_id:
+            send_analysis_result_email(analysis)
```

Comment:
Call this function in `process_impact_result` so it doesn't depend on web request.